fe/fe-core/src/main/java/com/starrocks/alter/MaterializedViewManager.java

```diff
@@ -0,0 +1,94 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.alter;
+
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
+import com.starrocks.catalog.MaterializedView;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.locks.ReentrantReadWriteLock;
+
+// a manager for materialized view
+// it has the following functions:
+// 1. manage the relations between manterialized view and base tables
+public class MaterializedViewManager {
+    // mv name -> materialized view
+    private Map<String, MaterializedView> materializedViews;
```

Comment:
why do need this key is MV name? when we rewrite the query, how do we use it?

Reply:
It is not very useful. I will remove it.
If needed to get mv from name, it can be done by Catalog